flit/wheel.py

```diff
@@ -8,6 +8,8 @@
 from . import common
 from . import inifile
 
+from shutil import copy2, copytree
```

Comment:
shutil is already imported above - for neatness, let's use `shutil.copy2` below and drop the extra import